htap.go

```diff
@@ -0,0 +1,8 @@
+package gosnowflake
```

Comment:
you don't create separate packages for the features?

Reply:
No, everything is in the `gosnowflake` package.